tests/helpers.mjs

```diff
@@ -158,3 +158,27 @@ export const setUpStake = async ({
     result: stakeResult,
   };
 };
+
+export const getBaseRegistrationFee = async ({ memory, timestamp }) => {
```

Comment:
nit: provide the name and then compute the length to return the name - becomes `getBaseRegistrtionFeeForName`

Reply:
0a253b6